### PR TITLE
Add PHP 8 compatibility and move to github actions

### PR DESCRIPTION
**Pull request by alexander-schranz**
See https://github.com/alexander-schranz/phpcr-shell/pull/1

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
```
Comment (dantleech) [inline .github/workflows/test-application.yaml:10]: i'd say remove these, as we won't be supporting separate branches.

Comment (alexander-schranz) [reply .github/workflows/test-application.yaml]: done. and tests are all green now: https://github.com/alexander-schranz/phpcr-shell/pull/1/checks

**Comment by dantleech** [comment]
Nice work :)

**Comment by dantleech** [comment]
Tagged 1.3.0 and updated the badge in the README :+1:

**Comment by alexander-schranz** [comment]
@dantleech Thats great, Thank you 🙏

**Pull request merged**